src/material-experimental/mdc-table/text-column.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {CdkTextColumn} from '@angular/cdk/table';
+import {ChangeDetectionStrategy, Component, ViewEncapsulation} from '@angular/core';
+
+/**
+ * Column that simply shows text content for the header and row cells. Assumes that the table
+ * is using the native table implementation (`<table>`).
+ *
+ * By default, the name of this column will be the header text and data property accessor.
+ * The header text can be overridden with the `headerText` input. Cell values can be overridden with
+ * the `dataAccessor` input. Change the text justification to the start or end using the `justify`
+ * input.
+ */
+@Component({
+  selector: 'mat-text-column',
+  template: `
+    <ng-container matColumnDef>
```

Comment:
For the other MDC components we also share the template string by exporting it through `material/table`. Can we do it for this component too?